src/wilcoxon.jl

```diff
@@ -161,7 +161,7 @@ function pvalue(x::ExactSignedRankTest; tail=:both)
     end
 end
 
-StatsBase.confint(x::ExactSignedRankTest, alpha::Real=0.05; tail=:both) = calculate_ci(x.vals, alpha; tail=tail)
+StatsBase.confint(x::ExactSignedRankTest; alpha::Real=0.05, tail=:both) = calculate_ci(x.vals, alpha, tail=tail)
```

Comment:
```suggestion
StatsBase.confint(x::ExactSignedRankTest; alpha::Real=0.05, tail=:both) = calculate_ci(x.vals, alpha; tail=tail)
```
Do we have a preference whether to use `;` (as below) or `,` in a function call?

Reply:
The package seems to be inconsistent in that regard, so I tried to keep the existing style.

Reply:
Yeah, I saw that, only this one appeared deliberate.

Reply:
There's another one just below... :-/

Reply:
FWIW two major style guides (YASGuide and BlueStyle) advise using `;` to separate keyword arguments. Even though I've worked with both on different codebases, I still find I use `,` out of habit...